packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SingleImageViewer/SingleImageViewerContainer.js

```diff
@@ -59,14 +48,22 @@ class SingleImageViewerContainer extends React.Component {
         loading: asyncStates.success
       })
     } catch (error) {
-      console.error(error)
-      this.setState({ loading: asyncStates.error })
+      this.onError(error)
     }
   }
 
+  onLoad (event) {
+    this.props.onReady(event)
+  }
+
+  onError (error) {
+    console.error(error)
+    this.setState({ loading: asyncStates.error })
+  }
+
   render () {
     const { loadingState } = this.state
-    const { subject } = this.props
+    const { onReady, subject } = this.props
```

Comment:
What's onReady doing here? I don't see it being used.

Reply:
My bad. It's supposed to be passed down as the `onLoad` handler for the SVG image.

Reply:
I've rewritten this, and added a test.